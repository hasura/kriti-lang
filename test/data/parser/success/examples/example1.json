[
  null,
  true,
  false,
  1,
  1.5,
  "hello",
  "hello123",
  [1, null, true],
  {"foo": 1},
  {"foo": 1, "bar": { "baz": [1, true]}}
]
