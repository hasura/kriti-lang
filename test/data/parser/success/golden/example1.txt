Array
    [ Null
    , Boolean True
    , Boolean False
    , Number 1.0
    , Number 1.5
    , String "hello"
    , String "hello123"
    , Array
        [ Number 1.0
        , Null
        , Boolean True
        ]
    , Object
        ( fromList
            [
                ( "foo"
                , Number 1.0
                )
            ]
        )
    , Object
        ( fromList
            [
                ( "bar"
                , Object
                    ( fromList
                        [
                            ( "baz"
                            , Array
                                [ Number 1.0
                                , Boolean True
                                ]
                            )
                        ]
                    )
                )
            ,
                ( "foo"
                , Number 1.0
                )
            ]
        )
    ]